packages/engine/src/scene/components/ModelComponent.tsx

```diff
@@ -211,6 +231,9 @@ function ModelReactor() {
         animations: scene.animations
       })
     }
+
+    removeLoadingGeo()
```

Comment:
Super small, but if the file loader errors (404 or some server error), this will never be hit

Reply:
added a remove function in the error useEffect